bib/internal/utils/utils.go

```diff
@@ -9,3 +16,17 @@ func IsMountpoint(path string) bool {
 	c.Stdout = nil
 	return c.Run() == nil
 }
+
+// Synchronously invoke a command, writing its stdout to our stdout,
+// and gathering stderr into a buffer which will be returned in err
+// in case of error.
+func RunCmdSync(cmdName string, args ...string) error {
+	logrus.Debugf("Running: %s %s", cmdName, strings.Join(args, " "))
+	cmd := exec.Command(cmdName, args...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("error running %s %s: %s: %w", cmdName, strings.Join(args, " "), err)
```

Comment:
The docstring mentions gathering stderr into a buffer but the code isn't doing it.  And there's a missing arg in the errorf which I guess should have been the buffered stderr?

Reply:
Thanks, fixed.  I had this locally but had forgotten to push apparently.
(Bigger picture juggling working on this and some other things, trying to use a Mac as a devenv, but was concurrently working on bugs like https://github.com/containers/podman/issues/21160 and have been concurrently destroying/recreating my podman machine.  And actually before this patch *and* before https://github.com/osbuild/bootc-image-builder/commit/c679e847d9d3b9334b3e4abfc18b8f6b1cfb7685#diff-6fd0e644fac1627088d13a0bdd084daee6f0ce1cca74d7343cdb0ffe0d87969cR91, bib happened to build for Mac)